src/Umbraco.Core/EmbeddedResources/Snippets/MultinodeTree-picker.cshtml

```diff
@@ -1,6 +1,4 @@
 @using Umbraco.Cms.Core.Models.PublishedContent
-@using Umbraco.Cms.Core.Routing
-@using Umbraco.Extensions
```

Comment:
Not sure we want to change this snippet, since some might be using the namespaces, in particular `Umbraco.Extensions`